src/lib/types/restrictions.ts

```diff
@@ -0,0 +1,98 @@
+export const PROCESS_ID_LENGTH: number = 32
+
+type TokenMetadataKey = string
+type TokenId = number
+type File = string
+type MetadataValue = File | string | TokenId | null
+
+enum MetadataValueType {
```

Comment:
It would be better if you could not use ENUM in ts https://www.aaron-powell.com/posts/2020-05-27-the-dangers-of-typescript-enums/

Ideally you would create a type e.g.


```ts
type Role = Owner | Customer | ....
```

You get the idea. Going to request changes just mainly because of this

Reply:
changed to not use `enums`